armi/bookkeeping/db/database3.py

```diff
@@ -77,7 +77,9 @@
 from armi.utils.textProcessors import resolveMarkupInclusions
 
 ORDER = interfaces.STACK_ORDER.BOOKKEEPING
-DB_VERSION = "3.1"
+DB_VERSION = "3.2"
```

Comment:
(your idea): Time for a version log?

Reply:
Added some docstring talk about versions, and a little changelog